ports/mimxrt10xx/README.md

```diff
@@ -49,6 +49,9 @@ Double tap to enter bootloader mode, then simply drag & drop `update-tinyuf2_BOA
 - [MIMX RT1010 Evaluation Kit](https://www.nxp.com/design/development-boards/i.mx-evaluation-and-development-boards/i.mx-rt1010-evaluation-kit:MIMXRT1010-EVK)
 - [MIMX RT1020 Evaluation Kit](https://www.nxp.com/design/development-boards/i.mx-evaluation-and-development-boards/i.mx-rt1020-evaluation-kit:MIMXRT1020-EVK)
 - [MIMX RT1060 Evaluation Kit](https://www.nxp.com/design/development-boards/i.mx-evaluation-and-development-boards/mimxrt1060-evk-i.mx-rt1060-evaluation-kit:MIMXRT1060-EVK)
+- [Teensy 4.0](boards/teensy40)

```

Comment:
must linked to manufacturer url

Reply:
> thank you for the PR, please check out review for suggested changes. Also what is the use of **nxp_blhost_sdphost** submodule ? if not needed, we should removed that

Oops, accidentally left that in.  Removed via 6031ec2ddd5b4db11dd9a5c7b8bb3aa7699b3673